pkg/server/server_update.go

```diff
@@ -110,7 +110,7 @@ func (s *Server) upgradeStatus(ctx context.Context) (bool, error) {
 		if newVersion == "" {
 			newVersion = version
 		} else if version != newVersion {
-			return false, errors.New("not all nodes are running the latest version yet")
+			return false, errors.Errorf("not all nodes are running the latest version yet (saw %s and %s)", newVersion, version)
```

Comment:
maybe import `cockroachdb/errors` while you're at it? (optionally also use `errors.Newf`)

Reply:
Will do next time I touch these files

Reply:
Actually there's a merge conflict, so I might as well do it!